pom.xml

```diff
@@ -66,6 +66,25 @@
             <id>release</id>
             <build>
                 <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-checkstyle-plugin</artifactId>
+                        <dependencies>
+                            <dependency>
+                                <groupId>com.puppycrawl.tools</groupId>
+                                <artifactId>checkstyle</artifactId>
+                                <version>8.31</version>
+                            </dependency>
+                            <dependency>
```

Comment:
Why is Java SDK added over here ?

Reply:
this runs checkstyle check during release
but including it in build only would be enough I guess.

Reply:
we use profiles to change default build config . As you have already added plugins in <build> . I don't think there is a need to mention them again in profile . Please check this once and remove plugin from profile